lexicons/app/bsky/feed/getFeedSuggestions.json

```diff
@@ -0,0 +1,31 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.getFeedSuggestions",
```

Comment:
Only thing I'm thinking about now is that I like `getSuggestedFeeds` better than the other way around 🤔

Reply:
i agree 👍